### PR TITLE
Distirbutions and random sampling

### PR DESCRIPTION
**Pull request by lilyclements**
Ready to be merged

### CODE REVIEW

File: instat/ucrDistributions.vb [hunk 1]
```diff
@@ -316,8 +316,8 @@ Public Class ucrDistributions
         clsBetaDist.strQFunctionName = "qbeta"
         clsBetaDist.strDFunctionName = "dbeta"
         clsBetaDist.bIsContinuous = True
-        clsBetaDist.AddParameter("shape1", "a", 1)
-        clsBetaDist.AddParameter("shape2", "b", 1)
+        clsBetaDist.AddParameter("shape1", "Shape", 1)
+        clsBetaDist.AddParameter("shape2", "Shape", 1)
```
Comment (dannyparsons) [inline instat/ucrDistributions.vb]: Shouldn't these have different names if they are different parameters? Even "Shape 1" and "Shape 2"?

File: instat/ucrDistributions.vb [hunk 2]
```diff
@@ -254,7 +254,7 @@ Public Class ucrDistributions
         clsWeibullDist.strQFunctionName = "qweibull"
         clsWeibullDist.strDFunctionName = "dweibull"
         clsWeibullDist.bIsContinuous = True
-        clsWeibullDist.AddParameter("shape", "Shape")
+        clsWeibullDist.AddParameter("shape", "Shape", 1)
```
Comment (lilyclements) [inline instat/ucrDistributions.vb]: I did, but I can see if we wouldn't.
We need that parameter to have a value, and while there isn't a default on R, I saw that there wasn't a default for 'probs' in geometric distribution on R either, and that had been given a value regardless. So, I added it in for consistency mainly!

Comment (dannyparsons) [inline instat/ucrDistributions.vb]: Ok, that's fine, just checking

File: instat/ucrDistributions.vb (hunk 1 above)
Comment (lilyclements) [inline instat/ucrDistributions.vb]: Oh yes, of course! I've done that now.

**Pull request merged**